ruby/vampires.rb

```diff
@@ -0,0 +1,47 @@
+puts "How many employees will be processed?"
+employee_number = gets.chomp
+n = employee_number.to_i
+n.times do
+  puts "What's your name?"
+  employee_name = gets.chomp
+  puts "How old are you? What year were you born? (right/ wrong)"
+  age = gets.chomp
+  puts "Our company cafeteria serves garlic bread. Should we worder some for you (y/n)?"
+  garlic_bread = gets.chomp
+  puts "Would you like to enroll in the company's health insurance (y/n)?"
+  insurance = gets.chomp
+
+  if employee_name == "Drake Cula" || employee_name == "Tu Fang"
```

Comment:
I like your use of setting a result and then printing it at the end.  Since the question asked us not to change the order of these prompts we could put each in an individual if block and then set the result for each scenario liek so.

```result = 'Results inconclusive'
if some_condition_1
result = 'thing_1'
end

if some_condition_2
result = `thing_2`
end

etc...
```